app/controllers/product_access_groups_controller.rb

```diff
@@ -16,6 +16,7 @@ def initialize
   end
 
   def index
+    @product = @instrument # required for tabnav_product
```

Comment:
Have you verified the other actions on this controller? It seems like this might live better as a before_action.

It's probably not worth refactoring this controller/views to use `@product` right now, but that should probably be the better longer-term solution.